script.js

```diff
@@ -208,3 +208,39 @@ window.onload = () => {
     });
   });
 };
+
+// form validation
+function validation() {
+  const email = document.getElementById('email').value;
+  const errorMessage = document.getElementById('error');
+  errorMessage.style.display = 'block';
+  if (email !== String(email).toLowerCase()) {
+    const text = 'Please enter a valid email!';
```

Comment:
- [ ] Making the error message more descriptive eg by telling them to make sure that their email is lowercase will help them.